api/src/main/java/com/codeforcommunity/dto/team/UsersTeamDataResponse.java

```diff
@@ -0,0 +1,41 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.enums.TeamRole;
+
+public class UsersTeamDataResponse {
```

Comment:
Either here or in the processor implementation when instantiating this class we need a TODO and a ticket for changing the fields of this class to userId and username as it is in the API specification https://docs.c4cneu.com/sftt/sftt-api-spec/#get-applicants

Reply:
done